rustls/src/msgs/fragmenter.rs

```diff
@@ -53,8 +60,9 @@ impl MessageFragmenter {
         }
     }
 
-    /// `max_fragment_size` is the maximum fragment size that will be produced --
-    /// this includes overhead. A `max_fragment_size` of 10 will produce TLS fragments
+    /// Set the maximum fragment size that will be produced.
+    ///
+    /// This includes overhead. A `max_fragment_size` of 10 will produce TLS fragments
```

Comment:
"fragments up to 10 bytes" -- (non-native speaker) does that need an "of" in there somewhere?

Reply:
Good point. Change to "will produce TLS fragments up to 10 bytes long."

I also took the opportunity to name the parameter `max_fragment_size` and to additionally document the meaning of `None`.